### PR TITLE
add  maja capslock indicator

### PR DESCRIPTION
**Pull request by moyi4681**
add  maja capslock indicator

### CODE REVIEW

File: keyboards/kbdfans/maja/maja.c [hunk 1]
```diff
@@ -102,3 +102,12 @@ void suspend_wakeup_init_kb(void) {
     rgb_matrix_set_suspend_state(false);
     suspend_wakeup_init_user();
 }
+
+__attribute__ ((weak))
+void rgb_matrix_indicators_user(void)
```
Comment (drashna) [inline keyboards/kbdfans/maja/maja.c:107]: Is this in the keymap, as well? 

If not, I think it should be, just in case.

Comment (moyi4681) [reply keyboards/kbdfans/maja/maja.c:107]: it's not in the keymap.

**Review by drashna — APPROVED** [review]

File: keyboards/kbdfans/maja/maja.c [hunk 2]
```diff
@@ -102,3 +102,12 @@ void suspend_wakeup_init_kb(void) {
...
+{
+    if (IS_HOST_LED_ON(USB_LED_CAPS_LOCK))
```
Comment (fauxpark) [inline keyboards/kbdfans/maja/maja.c:109]:
```suggestion
    if (host_keyboard_led_state().caps_lock)
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by fauxpark — APPROVED** [review]

**Comment by drashna** [comment]
Thanks!

**Pull request merged**